scripts/testdata/satellite-config.yaml.lock

```diff
@@ -334,6 +331,24 @@ identity.key-path: /root/.local/share/storj/identity/satellite/identity.key
 # number of update requests to process per transaction
 # overlay.update-stats-batch-size: 100
 
+# payments provider to use
+# payments.provider: ""
+
+# amount of time we wait before running next account balance update loop
+# payments.stripe-coin-payments.account-balance-update-interval: 1h30m0s
```

Comment:
Can we confirm those intervalls are ok with the release team?

Reply:
`chore` is disabled